src/Event/ConstructEvent.php

```diff
@@ -8,21 +7,23 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace WyriHaximus\TwigView\Event;
 
 use Cake\Event\Event;
+use Twig\Environment;
 use WyriHaximus\TwigView\View\TwigView;
 
 class ConstructEvent extends Event
 {
     const EVENT = 'TwigView.TwigView.construct';
 
     /**
-     * @param TwigView $twigView
-     * @param \Twig_Environment $twig
+     * @param  TwigView          $twigView
+     * @param  \Twig\Environment $twig
```

Comment:
Could be replaced with just `Environment` as it's imported at the top. Would prefer to do that with all classes used in docblocks

Reply:
It is a lot of work so please confirm this: 
I prefer FQNS in docblock as does cakephp/cakephp - without using an IDE this is much much better.

Reply:
Don't have any hard feelings for this so feel free to skip this. It's just a preference :smile:

Reply:
Doc blocks should always have FQCN, they must be independent from the code, even if use statements are already present.